docs/guides/install-qiskit.mdx

```diff
@@ -226,146 +226,6 @@ Additionally, Qiskit only supports the CPython implementation of the Python lang
   - Windows 10 32-bit
 </details>
 
-## Qiskit versioning
```

Comment:
Idea: you can avoid removing this content for now so that it is safe to merge this PR today. That will allow us to reduce the scope of the final PR we land right before the migration, so that it is only removing this duplicate content.

The downside is duplicating the content for a couple of days, which risks them falling out of sync.